src/main/java/com/selenium/course/pages/NewProductForm.java

```diff
@@ -0,0 +1,100 @@
+package com.selenium.course.pages;
+
+import com.selenium.course.framework.DriverManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebDriverException;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.CacheLookup;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.util.concurrent.TimeUnit;
+import static com.selenium.course.common.CommonUIMethods.check;
+import static com.selenium.course.common.CommonUIMethods.uncheck;
+
+/**
+ * Created by Andrea Castro on 6/12/2015.
+ */
+public class NewProductForm {
+
+    private WebDriver driver;
+    private WebDriverWait wait;
+
+
+    @FindBy(xpath = "//h1[contains(.,'Product Edit:')]")
+    @CacheLookup
+    WebElement ProductTitle;
+
+
+    @FindBy(id = "Name")
+    @CacheLookup
+    WebElement productNAmeField;
+
+    @FindBy(id = "ProductCode")
+    @CacheLookup
+    WebElement productCodeField;
+
+    @FindBy(xpath = "//textarea[@id='Description']")
+    @CacheLookup
+    WebElement productDescriptionField;
+
+    @FindBy(name = "save")
+    @CacheLookup
+    WebElement saveBtn;
+
+
+
+
+    public NewProductForm() {
+        this.driver = driver;
+        wait = DriverManager.getInstance().getWait();
+        driver = DriverManager.getInstance().getDriver();
+        PageFactory.initElements(driver, this);
+        try {
+            wait.withTimeout(3, TimeUnit.SECONDS).until(
```

Comment:
Consider using the constants for TIMEOUTs created by @caverogustavo 
Apply it to your classes